dataplane/saiserver/acl.go

```diff
@@ -328,6 +328,11 @@ func (a *acl) CreateAclEntry(ctx context.Context, req *saipb.CreateAclEntryReque
 			},
 		})
 	}
+	if req.ActionRedirect != nil {
+		aReq.Actions = append(aReq.Actions,
+			fwdconfig.Action(fwdconfig.UpdateAction(fwdpb.UpdateType_UPDATE_TYPE_SET,
```

Comment:
should probably add a LOOKUP l2mc action here

Reply:
Done